shadows/framework/src/main/java/org/robolectric/shadows/ShadowCryptoObject.java

```diff
@@ -0,0 +1,60 @@
+package org.robolectric.shadows;
+
+import static android.os.Build.VERSION_CODES.P;
+import static android.os.Build.VERSION_CODES.R;
+import static android.os.Build.VERSION_CODES.TIRAMISU;
+
+import android.hardware.biometrics.CryptoObject;
+import android.security.identity.IdentityCredential;
+import android.security.identity.PresentationSession;
+import java.security.Signature;
+import javax.crypto.Cipher;
+import javax.crypto.Mac;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@SuppressWarnings("UnusedDeclaration")
+@Implements(value = CryptoObject.class, isInAndroidSdk = false, minSdk = P)
```

Comment:
It this entire shadow a no-op? Can it just use `@Implements(callThroughByDefault=false)`?

Reply:
Pretty much, yes. I was not aware of this functionality, will try it out

Reply:
Hmm, on second thought: No. We don't want to rely on the default implementation as it makes heavy use of an object of a class that does not exist. So best here is to stub with null returns, not rely on the real implementation

Reply:
More details: https://github.com/robolectric/robolectric/pull/8259#discussion_r1231501304

Reply:
Oh, it looks better.